Terminal.Gui/App/Application.Run.cs

```diff
@@ -212,8 +212,15 @@ public static RunState Begin (Toplevel toplevel)
 
         NotifyNewRunState?.Invoke (toplevel, new (rs));
 
-        // Force an Idle event so that an Iteration (and Refresh) happen.
-        Invoke (() => { });
+        if (!ConsoleDriver.RunningUnitTests)
+        {
+            // Force an Idle event so that an Iteration (and Refresh) happen.
+            Task.Run (() =>
+                      {
+                          Invoke (() => { });
+                          Task.Delay (10).Wait ();
```

Comment:
Why 10ms. Why not 100? Why not 1? 

Can this be done in a way where someone reading the code understands precisely how this is working?

Writing a  // comment may work.